subgraph/src/mint.ts

```diff
@@ -68,13 +75,37 @@ export function handleNewMint(event: NewMintEvent): void {
   token.accessPointAutoApproval = accessPointAutoApproval;
   token.owner = ownerAddress;
   token.verifier = verifierAddress;
+  token.verified = false;
   token.mintTransaction = event.transaction.hash.concatI32(
     event.logIndex.toI32()
   );
   token.mintedBy = event.params.minter;
   token.controllers = [ownerAddress];
+  token.createdAt = event.block.timestamp;
+
+  // Populate GitRepository entity
+  let repository = GitRepository.load(gitRepository);
+  if (!repository) {
+    repository = new GitRepository(gitRepository);
+  }
+
+  let repositoryTokens = repository.tokens;
+  if (repositoryTokens === null) {
+    repositoryTokens = [token.id];
+  } else {
+    repositoryTokens.push(token.id);
+  }
+  repository.tokens = repositoryTokens;
+
+  // Increase total tokens counter
+  const collection = Collection.load(CollectionId);
+  if (collection) {
+    collection.totalTokens = collection.totalTokens.plus(BigInt.fromU32(1));
+    collection.save();
+  }
 
   // Save entities
   owner.save();
   token.save();
+  repository.save();
```

Comment:
verifier should be saved too.

Reply:
Update on this from my discord chat with @zoruka :

Since the verifier is not created here, checking its existence will be enough.